presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -85,6 +85,7 @@
 
     private HiveStorageFormat hiveStorageFormat = ORC;
     private HiveCompressionCodec compressionCodec = HiveCompressionCodec.GZIP;
+    private HiveCompressionCodec orcCompressionCodec = HiveCompressionCodec.GZIP;
```

Comment:
I'm curious, why do we still set the default codec to be gzip for orc? As I saw from the note, ZSTD is better than GZIP in both compression ratio and CPU.

Reply:
@yingsu00 The plan is to roll out this change cluster by cluster. Thats why a config has been added in this PR.